zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -1084,7 +1090,6 @@ mod tests {
         let (account, _usk) = db_data
             .create_account(&Secret::new(seed.to_vec()), birthday)
             .unwrap();
-        assert_eq!(account, AccountId::ZERO);
```

Comment:
Replace this with a check that `zip32::AccountId::ZERO` is the account that was created.